dashboard/config/routes.rb

```diff
@@ -842,6 +842,7 @@
     get '/dashboardapi/v1/pd/workshops_as_facilitator_for_pl_page', to: 'pd/professional_learning#workshops_as_facilitator_for_pl_page'
     get '/dashboardapi/v1/pd/workshops_as_organizer_for_pl_page', to: 'pd/professional_learning#workshops_as_organizer_for_pl_page'
     get '/dashboardapi/v1/pd/workshops_as_program_manager_for_pl_page', to: 'pd/professional_learning#workshops_as_program_manager_for_pl_page'
+    get '/dashboardapi/v1/pd/regional_workshop_data/:zip_code', to: 'pd/professional_learning#regional_workshop_data'
```

Comment:
is there an opportunity here to use the new Rails routing guidelines? (In general, `dashboardapi` routes should be deprecated)

Reply:
https://docs.google.com/presentation/d/1Exim9a0NU05U4yFDPvEf2NmwFv-7qShlHsu7iGOh8So/edit

Reply:
Good call will do! I'll do a followup PR to cleanup the other `professional_learning_controller` routes